Src/Newtonsoft.Json/Bson/BsonReader.cs

```diff
@@ -224,6 +224,7 @@ public override bool Read()
 
         /// <summary>
         /// Changes the reader's state to <see cref="JsonReader.State.Closed"/>.
```

Comment:
Btw., working on this issue has made me wonder whether "Changes the reader's state to State.Closed" ought to be simplified to "Closes this reader" (like it is for the writer classes already).